lib/ts/normalisedURLPath.ts

```diff
@@ -43,7 +43,7 @@ export default class NormalisedURLPath {
     };
 }
 
-function normaliseURLPathOrThrowError(input: string): string {
+export function normaliseURLPathOrThrowError(input: string): string {
```

Comment:
You do not need to export this since the `NormalisedURLPath` class is already exported. Use that instead